benchmarks/addressable-getters/benchmark.rb

```diff
@@ -0,0 +1,21 @@
+require_relative "../../harness/loader"
+
+Dir.chdir __dir__
+use_gemfile
+
+require "addressable/uri"
+
+COMPLEX_URI = "https://user:pass@example.com:8080/path/to/resource?query=value&foo=bar#fragment"
+
+run_benchmark(100) do
+  100.times do
```

Comment:
I think we should a) bump the iteration count to ~10k per loop and b) use while loops because that has less per-loop overhead per than invokeblock and can more directly measure this otherwise quite small thing